.github/ISSUE_TEMPLATE/🐜bug-report--vs-code-runtime-extension.md

```diff
@@ -0,0 +1,26 @@
+---
+name: "\U0001F41Cbug report: vs-code runtime extension"
+about: 'bug report: vs-code runtime extension'
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+<!--
```

Comment:
In markdown, this becomes a comment, so it is not visible.  I reccommend using the quote and emoji technique :)

such as 
> :exclamation: Privacy alert! Blah

Which you can see by trying to edit my comment, it uses the > symbol and then the emoji name in :